app/src/main/java/ai/elimu/vitabu/ui/storybook/ChapterFragment.java

```diff
@@ -61,7 +61,7 @@
 
     private StoryBookChapterGson storyBookChapter;
 
-    protected String[] chapterText = {};
+    protected String[] chapterParagraph = {};
```

Comment:
And perhaps make the name plural, since it's an array?
```suggestion
    protected String[] chapterParagraphs = {};
```